manifests/cf-manifest/cloud-config/020-cf-vm-types.yml

```diff
@@ -0,0 +1,125 @@
+---
+meta:
+  default_env:
+    bosh:
+      password: (( grab secrets.vcap_password ))
+
+vm_types:
+  - name: small
+    network: cf
+    env: (( grab meta.default_env ))
+    cloud_properties:
+      instance_type: m3.medium
```

Comment:
On the 2nd look, we use only 7 smalls in the current deployment, potentially removing 2 of them in the future. We could switch to T2.small, which should still have 20% of CPU performance with no CPU credits - which looks like it's enough for the type of workload small instances do in our deployment. Added to improvements spreadsheet.